tests/perfalert/test_perf_sheriff_bot.py

```diff
@@ -27,7 +27,7 @@
 # -> up to here
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture()
 def record_context_sample():
```

Comment:
I think you need to move this fixture in `tests/perfalert/conftest.py`, otherwise pytest continues to complain it cannot find it.